cli/flags.go

```diff
@@ -53,6 +53,11 @@ var (
 		Value: "25s",
 		Usage: "period between each heartbeat",
 	}
+	flTTL = cli.StringFlag{
+		Name:  "time-to-live, ttl",
```

Comment:
Should we call this just `ttl`?

Also, what about removing `hb` - just `heartbeat`?

I think shortcut helpers don't make much sense for advanced options